definition/bin/upload-ccd-definition-to-env.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+set -e
```

Comment:
i dont actually think this will work 🙈 but...it should be here... we should make use of other Docker containers

Reply:
am working on above